src/tool/cppwinrt/cppwinrt/helpers.h

```diff
@@ -157,6 +157,22 @@ namespace xlang
         return is_remove_overload(method) || has_attribute(method, "Windows.Foundation.Metadata", "NoExceptionAttribute");
     }
 
+    static bool is_always_disabled(TypeDef const& type)
+    {
+        auto feature = get_attribute(type, "Windows.Foundation.Metadata", "FeatureAttribute");
+
+        if (!feature)
+        {
+            return false;
+        }
+
+        // The first argument is a Windows.Foundation.Metadata.FeatureStage enum class.
+        auto stage = std::get<ElemSig::EnumValue>(std::get<ElemSig>(feature.Value().FixedArgs()[0].value).value);
+
+        // FeatureStage::AlwaysDisabled is zero.
+        return std::get<int32_t>(stage.value) == 0;
```

Comment:
I *suppose* in this case, this is always true, but it's not exactly setting a great precedent for decoding enums from metadata without relying on hard-coded integer values. We've had some comments about usability of this library, and we already have nearly all the pieces in place to do this in a more reliable manner.

Something like this would do the trick:
```c++
// Added to ElemSig::EnumValue
bool equals_enumerator(std::string_view name) const
{
  // Perhaps these first few lines should be EnumDefinition::get_enumerator(std::string_view name)?
  auto fields = type.m_typedef.FieldList();
  auto field = std::find_if(begin(fields), end(fields), [name](Field const& f) { return f.Name() == name; });
  XLANG_ASSERT(field != end(fields));

  // EnumValue::value_type is a strict subset of Constant::constant_type - simple to convert.
  auto input_value = std::visit([](auto&& v) { return Constant::contant_type{ v }; }, value.value);
  return field.Constant().Value() == input_value;
}
```
This would better serve the library in the long term and immediately improve readability of these sorts of checks - cppwinrt's handling of ComposableAttribute would also benefit.